docs/source/changelog.md

```diff
@@ -12,6 +12,203 @@ and as we merge [breaking changes in pull
 requests](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pulls?q=is%3Apr+is%3Aclosed+label%3Abreaking),
 this list should be updated.
 
+## 2.0
+
+### 2.0.0
+
+#### Highlights
+
+- **TODO**
+
+  TODO
```

Comment:
```suggestion
Z2JH 2.0.0 is the first major release since 1.0.0 was released in June 2021, and contains major upgrades to all JupyterHub components.

JupyterHub 2.x includes new RBAC support allowing fine-grained access control to hub services and servers

JupyterLab, the next-generation Notebook interface, is now the default interface seen by users. This brings a full development environment with a large number of extensions developed by the Jupyter community.

This release also includes several smaller changes that help Z2JH interface better with the rest of the Jupyter community such as not overriding a Docker image's command, and using standard Helm chart parameter names to match with other Helm charts.
Although these are breaking changes they will greatly improve the maintainability of the JupyterHub chart in future, and should also make it easier for new users to get started.
```

Thought  might as well make a start now.